testcase 'Second testcase'
log 'fail on purpose'
has_text 'No such text'